api/v1/api.md

```diff
@@ -0,0 +1,217 @@
+# dcrtime API Specification
+
+## V1
+
+This document describes the REST API provided by a `dcrtimed` server.  This API allows users to create and upload hashes which are periodically submitted to the Decred blockchain.  It also provides the ability to to confirm such the addition of the hash to a timestamped collection and showing and validating their inclusion in the Decred blockchain.
+
+**Methods**
+
+- [`Timestamp`](#timestamp)
+- [`Verify`](#verify)
+
+**Return Codes**
+
+- [`ResultOK`](#ResultOK)
+- [`ResultExistsError`](#ResultExistsError)
+- [`ResultDoesntExistsError`](#ResultDoesntExistsError)
+- [`ResultDisabled`](#ResultDisabled)
+
+### Methods
+
+#### `Timestamp`
+
+Upload one or more digests to the time server.  The server will then add these digests to a collection and eventually to a transaction that goes in a Decred block.  This method only returns if the server was able to add the block to a collection, not if it has made it into a block (since that is done in batches at a set time interval that is not related to the api calls).
```

Comment:
"This method only returns if the server was able to add the block to a collection, not if it has made it into a block"

That is not true.  The call always returns immediately.  You can then query the digest and get information on it.  It'll then confirm if the digest has been anchored or not.